packages/common/src/index.ts

```diff
@@ -320,3 +320,25 @@ export const convertStatusToText = (
       return "PENDING";
   }
 };
+
+export const prefixZero = (i: number): string =>
```

Comment:
Can we make the name more indicative of what the function does? Maybe "padSingleDigitWithZero" or something like that?

Reply:
done